### ISSUE TITLE
Timing of addlifecycleRule execution

### ISSUE THREAD

**Issue by secmin** [googleapis/nodejs-storage#1247]
For example, do I just have to add the rule once when the server starts?
Or do I need to tie it to a bucket at file upload time?
The reason for the question is that I thought that tying it to the bucket at upload time would update the rule multiple times and perform an unnecessary update process.

**Comment by frankyn** [comment]
Hi @secmin 

Thanks for filing your question. Object Lifecycle Management rules should only need to be added once and are ran once a day. 

I recommend you set the OLM rules separate from your applications object upload.

Could you tell me a little bit more of what you're trying to do?

**Comment by secmin** [comment]
The problem is now solved because we have subdivided the buckets into a structure where the rules are managed in the console.
At the time, I was trying to change the rules programmatically, and I was afraid that the changes would not be updated if they were only executed once at instance time. So I thought I would need to update the rule changes at upload time.
However, my fears were unfounded, as the comments said it would run once a day.
